docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -0,0 +1,84 @@
+# Error Handling
+
+Error handlers allow for the connection to decide how to continue fetching data according to the contents of the response from the partner API. Depending on attributes of the response such as status code, text body, or headers, the connection can continue making requests, retry unsuccessful attempts, or fail the sync.
```

Comment:
When I think of connection, I think of a [source/destination pair + their config](https://docs.airbyte.com/understanding-airbyte/connections/). How about source or connector?

Reply:
yah connector is also more consistent w/ other documentation will fix this throughout